src/containers/DeveloperPortal/containers/SdkAndLibraries/components/ConfirmationModal/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {createPortal} from 'react-dom';
+import {Link} from 'react-router-dom';
+import useOnclickOutside from 'react-cool-onclickoutside';
+
+import {Button} from 'components';
+import {SFC} from 'types/generic';
+
+import LeaveSiteImg from '../../assets/leave-site.svg';
+
+import './ConfirmationModal.scss';
+
+type Props = {
+  url: string | null;
```

Comment:
`url?: string` would be ok?

Reply:
not sure, as the passed the `url` can have `string` or `null`; based on this I open the modal.

I thought `url?: string` works for `undefined`(more specifically, when a value is not passed in) and `string`? It works for `null` too?

Reply:
Yes `?` only accepts `undefined`

I see, in my personal opinion, if it is an optional prop, we should use `?` parameter, because to accommodate when the `url` is not passed in as props. If you set it to `string | null`, then a url needs to be always passed in.